### PR TITLE
[Keyboard] Adding Gerald65

### PR DESCRIPTION
**Pull request by PaperCraneKeyboards**
## Description

Creation of the Gerald65 keyboard with a default keymap

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/papercranekeyboards/gerald65/config.h [hunk 1]
```diff
@@ -0,0 +1,138 @@
+// Copyright 2021 PaperCraneKeyboards (@PaperCraneKeyboards)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0x1003 // P,C (Paper Crane)
```
Comment (drashna) [inline keyboards/papercranekeyboards/gerald65/config.h:9]: https://www.the-sz.com/products/usbid/index.php?v=0x1003&p=&n=

This Vendor Id is in use, and needs to be changed. 

Anything above 0x4000 is relatively safe.

File: keyboards/papercranekeyboards/gerald65/readme.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+![gerald65](https://imgur.com/a/G2VuJoD)
+
+A 65% ergonomic acrylic keyboard emphasizing customizability.
+
+* Keyboard Maintainer: [PaperCraneKeyboards](https://github.com/PaperCraneKeyboards)
+* Hardware Supported: Gerald65 PCB
+* Hardware Availability: papercranekeyboards.com
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/papercranekeyboards/gerald65/readme.md:11]: Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/papercranekeyboards/gerald65/config.h (hunk 1 above)
Comment (PaperCraneKeyboards) [reply keyboards/papercranekeyboards/gerald65/config.h]: Oh thank you! Changed to 0x5012.

File: keyboards/papercranekeyboards/gerald65/readme.md (hunk 2 above)
Comment (PaperCraneKeyboards) [reply keyboards/papercranekeyboards/gerald65/readme.md:11]: Added those instructions with specifics for this PCB. Thanks!

**Comment by aidansmith459** [comment]
@drashna requested changes have been fixed. Would it be possible to get this checked again? Thanks! :)

**Comment by PaperCraneKeyboards** [comment]
@drashna requested changes have been fixed. Any chance we could get this checked again? Thanks so much!

**Review by drashna — APPROVED** [review]

File: keyboards/papercranekeyboards/gerald65/config.h [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+//#define NO_ACTION_TAPPING
+//#define NO_ACTION_ONESHOT
+
+/* disable these deprecated features by default */
+#define NO_ACTION_MACRO
+#define NO_ACTION_FUNCTION
+
+/* Bootmagic Lite key configuration */
+//#define BOOTMAGIC_LITE_ROW 0
+//#define BOOTMAGIC_LITE_COLUMN 0
```
Comment (noroadsleft) [inline keyboards/papercranekeyboards/gerald65/config.h:138]: `NO_ACTION_MACRO` and `NO_ACTION_FUNCTION` don't do anything anymore.

`BOOTMAGIC_LITE_ROW` and `BOOTMAGIC_LITE_COLUMN` only need to be defined if you're using a location other than Row 0 Column 0, which is [the default set by QMK Firmware](https://github.com/qmk/qmk_firmware/blob/04826d1aa175c5aeb808ae9af8d90d164532f199/quantum/bootmagic/bootmagic_lite.h#L18-L23).

```suggestion
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**